en/orm/behaviors.rst

```diff
@@ -242,6 +242,29 @@ rename/remove finder methods when adding a behavior to a table. For example::
         ]
     ]);
 
+Defining Custom Marshalling Behavior Logic
+==========================================
```

Comment:
I'd name this `Transforming request data into Entity properties` for easier search